spec/factories.rb

```diff
@@ -1,2 +1,7 @@
 FactoryGirl.define do
+  factory :user do
+    sequence(:email) { |i| "user#{i}@example.com" }
+    password "password"
+    zipcode { "80221" }
```

Comment:
Do you mean to ask why it's lazy?

The answer is "I don't know, it doesn't need to be"